test/integration/policy_psp_test.go

```diff
@@ -0,0 +1,150 @@
+// Copyright (c) 2022 Red Hat, Inc.
+// Copyright Contributors to the Open Cluster Management project
+
+package integration
+
+import (
+	"context"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	policiesv1 "github.com/stolostron/governance-policy-propagator/api/v1"
+	"github.com/stolostron/governance-policy-propagator/test/utils"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	k8stypes "k8s.io/apimachinery/pkg/types"
+
+	"github.com/stolostron/governance-policy-framework/test/common"
+)
+
+var _ = Describe("GRC: [P1][Sev1][policy-grc] Test the policy-psp policy", Label("policy-collection", "stable"), func() {
+	const (
+		rootPolicyName   = "policy-podsecuritypolicy"
+		rootPolicyURL    = policyCollectSCURL + "policy-psp.yaml"
+		rootPolicyNSName = "policy-psp"
+		pspName          = "sample-restricted-psp"
+	)
+
+	It("stable/"+rootPolicyName+" should be created on the hub cluster", func() {
+		By("Creating " + rootPolicyName + " on the hub")
+		output, err := utils.KubectlWithOutput(
```

Comment:
```suggestion
		_, err := utils.KubectlWithOutput(
```
The output is already printed in `utils.KubectlWithOutput`. So you can just ignore the return value.